### PR TITLE
Fix writing style on User Documentation (Home)

### PR DESCRIPTION
**Pull request by MinhHuong**
Fix some parts of the page Home: 
- Add a heading for every part (at first the page Home began with the video, I add the heading **Ideas**, so structure now becomes Ideas - Features - Requirements - Contributing to Oppia - Helpful links)
- Rewrite the features so it has a homogenous style (all begin with a base form verb)
- Minor fix (italics, bold, etc.)

### CODE REVIEW

File: pages/TheExplorationEditor.html [hunk 1]
```diff
@@ -38,8 +38,8 @@
   </h3>
 
   <p>
-    A card consists of three parts: some 'content', an 'interaction' (or
-    question), and 'responses' that specify what Oppia should do next, based on
+    A card consists of three parts: some <em><strong>content</strong></em>, an <em><strong>interaction</strong></em> (or
```
Comment (seanlip) [inline pages/TheExplorationEditor.html]: Per discussion in #1614, I thought we were going for just 'strong' here (no 'em')?

File: pages/home.html [hunk 2]
```diff
@@ -1,15 +1,21 @@
...
+    Ideas
+  </h2>
   <div class="video-container">
     <iframe title="Introductory video for Oppia" width="520" height="350"
             src="https://www.youtube.com/embed/Ntcw0H0hwPU" frameborder="0"
             allowfullscreen style="display: block; margin: 0 auto 40px auto;">
     </iframe>
   </div>
-
+  <p></p>
```
Comment (seanlip) [inline pages/home.html]: Why the empty paragraph? If you want a semantic break, use `<br>` instead.

File: pages/WhatIsOppia.html [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
   <p>
     Oppia is an online learning tool that enables anyone to easily create and
-    share interactive activities ('explorations') which try to recreate the
+    share interactive activities ("explorations") which try to recreate the
```
Comment (seanlip) [inline pages/WhatIsOppia.html]: This is a term of art, so per our discussion, should it be single quotes instead?

File: pages/home.html [hunk 4]
```diff
@@ -1,15 +1,21 @@
 <section id="main_content" class="inner">
 
+  <h2>
+    <a id="main-idea" class="anchor" href="#main-idea" aria-hidden="true">
+      <span class="octicon octicon-link"></span>
+    </a>
+    Ideas
```
Comment (seanlip) [inline pages/home.html]: This seems a bit of an odd header...

File: pages/home.html [hunk 5]
```diff
@@ -62,15 +66,17 @@
...
-  <ul>
-    <li>Simulates a 1-on-1 conversation with a tutor.</li>
-    <li>Able to give targeted feedback to learners based on their answers.</li>
-    <li>Allows learners to leave feedback on explorations.</li>
-    <li>Explorations are embeddable in any webpage.</li>
-    <li>Explorations can be played on mobile devices (except for some of the more complex interactions).</li>
-    <li>The exploration editor supports full version history, multiple authors, parameters and analytics dashboards.</li>
-    <li>There are extension frameworks for building new interactions and classification rules.</li>
-  </ul>
+  <p>With Oppia, you can:
```
Comment (seanlip) [inline pages/home.html:66]: +1, I like this change.

File: pages/home.html [hunk 6]
```diff
@@ -62,15 +66,17 @@
...
+    <ul>
+      <li>simulate a 1-on-1 conversation with a tutor,</li>
+      <li>give targeted feedback to learners based on their answers (tutors),</li>
```
Comment (seanlip) [inline pages/home.html]: I don't understand the point of '(tutors)' here and '(learners)' in the next bullet point.

File: pages/home.html [hunk 7]
```diff
@@ -62,15 +66,17 @@
...
+      <li>leave feedback on explorations (learners),</li>
+      <li>embed explorations in any webpage.</li>
```
Comment (seanlip) [inline pages/home.html]: comma at end? Or just remove the final punctuation mark in all cases?

File: pages/home.html [hunk 8]
```diff
@@ -62,15 +66,17 @@
...
+      <li>play explorations on mobile devices (except for some of the more complex interactions),</li>
+      <li>have support on full version history, multiple authors, parameters and analytics dashboards in the exploration editor,</li>
+      <li>build new interactions and classification rules using extension framework.</li>
```
Comment (seanlip) [inline pages/home.html]: Actually, let's scrap this last point. It's developer-focused, as opposed to the others, which are end-user-focused.

File: pages/home.html [hunk 9]
```diff
@@ -21,20 +27,18 @@
     Bite-sized
   </h3>
   <p>
-    Explorations are easy to create and quick to play. They can be published in
-    the exploration gallery, shared on social media, or embedded on your
-    webpage.
+    Easy to create and quick to play, explorations can also be published in the exploration gallery, shared on social media, or embedded on your webpage.
```
Comment (seanlip) [inline pages/home.html]: I don't know if I like the inversion here -- it feels a bit less direct than what it originally was. Do you prefer this version?

File: static/css/stylesheet.css [hunk 10]
```diff
@@ -159,9 +160,12 @@ h1 {
 }
 
 h2 {
-  padding-bottom: 10px;
+  padding-bottom: 0px;
```
Comment (seanlip) [inline static/css/stylesheet.css]: We try to omit units when the value is 0 -- so, just "padding-bottom: 0"

File: pages/TheExplorationEditor.html (hunk 1 above)
Comment (MinhHuong) [inline pages/TheExplorationEditor.html]: Ah my fault, I made an example to show you on the thread of issue #1614, then I came back to fix the Home page and forgot to reset this back back to strong. Sorry!

File: pages/home.html (hunk 2 above)
Comment (MinhHuong) [inline pages/home.html]: Okie I'll fix it.

File: static/css/stylesheet.css [hunk 11]
```diff
@@ -159,9 +160,12 @@ h1 {
...
+  margin-bottom: 20px;
   font-size: 32px;
-  background: url('../images/bg_hr.png') repeat-x bottom;
+  /*background: url('../images/bg_hr.png') repeat-x bottom;*/
```
Comment (seanlip) [inline static/css/stylesheet.css]: OK to just remove both lines altogether -- we don't check in commented-out code. It's already in the version history, anyway.

File: pages/home.html (hunk 6 above)
Comment (MinhHuong) [inline pages/home.html]: Yeah it's a bit ambiguous, the tutors in (..) means "you can ... _as a tutors_", I'll fix to make it clearer

File: pages/home.html [hunk 12]
```diff
@@ -100,10 +106,8 @@
     Contributing to Oppia
   </h2>
   <p>
-    Oppia is built by the community for the community. Our mission is to enable
-    anyone to learn anything they want to in an effective, enjoyable way, and
-    building a platform that allows anyone to create and share bite-sized
-    interactive learning material is just a first step.
+    <em>Built by the community for the community</em> – Oppia aims to enable
```
Comment (seanlip) [inline pages/home.html]: The inversion here feels a bit weird to me, too. Is the italicized text meant to be a headline?

What is the purpose of the change?

File: pages/home.html (hunk 4 above)
Comment (MinhHuong) [inline pages/home.html]: Do you mean the content (Ideas) or the structure of the code?

Comment (seanlip) [inline pages/home.html]: The content.

File: pages/home.html (hunk 7 above)
Comment (MinhHuong) [inline pages/home.html]: This part is originally a paragraph:

_With Oppia, you can simulate a 1-on-1 conversation with a tutor, give targeted feedback to learners, ..... ._

But instead of giving readers a big paragraph, we break it into bullet points. Comma at end of each idea means that there are still more ideas below, and the last bulle point is marked by a period (to end the whole paragraph).

This is the rule for citing ideas indicated as an ISO standards. I learnt it while writing a report for school :smile:

File: pages/home.html (hunk 8 above)
Comment (MinhHuong) [inline pages/home.html]: Okie, noted. Also, could we clarify the _full version history_ term (in the bullet point before the last one)? I don't really understand its meaning. Could we choose another expressions that are easier to understand?

File: pages/home.html (hunk 7 above)
Comment (seanlip) [inline pages/home.html]: OK, that sounds reasonable to me. This means that this particular bullet point should have a comma at the end, though.

File: pages/home.html (hunk 8 above)
Comment (seanlip) [inline pages/home.html]: "Full version history" means that you can see and go back to previous versions. Here's the Wikipedia analogue: [link](https://en.wikipedia.org/w/index.php?title=Florida_Keys&action=history).

Good call on this not being clear; feel free to reword it.

File: pages/WhatIsOppia.html (hunk 3 above)
Comment (MinhHuong) [inline pages/WhatIsOppia.html]: I made a change as example on this page and forgot to reset it... Oh but this is a technical/new word, should it be bold instead of single-quoted ?

Comment (seanlip) [inline pages/WhatIsOppia.html]: Maybe "... share interactive activities, called **explorations**, which try to ..."?

Comment (MinhHuong) [inline pages/WhatIsOppia.html]: Or should we say "...share **explorations** (interactive activities) which try to recreate ...", so the term **explorations** attract more attention from readers?

File: pages/home.html (hunk 7 above)
Comment (MinhHuong) [inline pages/home.html]: Ah okie I see. Yes it should be a comma at that bullet point instead of a period. My fault!

File: pages/home.html (hunk 9 above)
Comment (MinhHuong) [inline pages/home.html]: I feel like the 2 original sentences can be rephrased to be one sentence only, and it still has the full meaning as the original one. I think the less we have to read, the more we're able to remember.

File: static/css/stylesheet.css (hunk 10 above)
Comment (MinhHuong) [inline static/css/stylesheet.css]: Okie, noted.

File: pages/home.html (hunk 12 above)
Comment (MinhHuong) [inline pages/home.html]: I want to insist on this spirit, I think the expression _Built by the community for the community_ is like a motto, a principal property of Oppia. It is too long to be a headline though. It can create a strong impression on readers when they see this motto first, and read the next content later.

File: pages/home.html (hunk 4 above)
Comment (MinhHuong) [inline pages/home.html]: The structure of the page is divided to 5 main sections:
- Section 1 with the video, bite-sized, interactive, adaptive
- Section 2: Features
- Section 3: Requirements
- Section 4: Contributing to Oppia
- Section 5: Helpful Links (should we change it to sentence capitalization 'Helpful links'?)

I observe that each section has its own heading, except the first one where we talk about the creative ideas behind Oppia. We should give it a heading too. Because once readers see a heading, their mind automatically predict the content of that section, it'll help them not to get lost in the documentation.

File: pages/WhatIsOppia.html (hunk 3 above)
Comment (seanlip) [inline pages/WhatIsOppia.html]: I think I prefer the first one better (i.e., "interactive activities" outside the brackets) since that's more understandable to new users. But I don't feel too strongly.

File: pages/home.html (hunk 4 above)
Comment (seanlip) [inline pages/home.html]: How about "Motivation"?

File: pages/home.html (hunk 9 above)
Comment (seanlip) [inline pages/home.html]: But the resulting sentence is long and combines lots of ideas. I think it feels a bit clunky.

(Speaking of understandability -- did you user test? It might be worth doing that with some of your friends, or folks on the gitter chat.)

File: pages/home.html (hunk 12 above)
Comment (seanlip) [inline pages/home.html]: I like it as a motto, but we don't really use this construct (headline in italics) anywhere else on the page.

How about putting it in italics? "Oppia is _built by the community for the community_ -- our mission is to ..."

File: pages/home.html (hunk 4 above)
Comment (MinhHuong) [inline pages/home.html]: Ah great! I was looking for the right word but couldn't find it. Thanks :+1:

File: pages/home.html (hunk 9 above)
Comment (MinhHuong) [inline pages/home.html]: I've just asked my friends to see which version they prefer :smile:

File: pages/WhatIsOppia.html (hunk 3 above)
Comment (MinhHuong) [inline pages/WhatIsOppia.html]: Okie, so maybe it'll be like this: 

Oppia is an online learning tool that enables anyone to easily create and share interactive activities (or **explorations**) which try to recreate

File: pages/home.html (hunk 12 above)
Comment (MinhHuong) [inline pages/home.html]: Or should we just remove the italics, so it would be simply:

"Built by the community for the community, Oppia aims to..."

I think it would be shoter, and still gramatically correct.

File: pages/home.html (hunk 9 above)
Comment (seanlip) [inline pages/home.html]: Sounds good! You also mentioned bringing it to the oppia-chat channel -- yep, feel free to do that, that's totally fine!

File: pages/home.html (hunk 12 above)
Comment (seanlip) [inline pages/home.html]: That seems fine too.

File: pages/home.html (hunk 9 above)
Comment (MinhHuong) [inline pages/home.html]: I've just collected my friends' opinions on this subject, quite interesting! Some say the original version (with 2 sentences) is clean and clear, good to go if we're writing a report. Others say the 2nd version is impressive, easy to understand, and catchy.

I collect all these opinions from my Vietnamese friends. The interesting part is that those who live in Vietnam like the second version, meanwhile those who live/study abroad prefer the original one.

Comment (seanlip) [inline pages/home.html]: Interesting! In that case, I think it makes sense to go with the original one, since a more diverse group of people prefer it. Would you agree?

Comment (MinhHuong) [inline pages/home.html]: Totally agree. I've also brought this question to the oppia-chat and all of the responses incline to the original version. I focused too much on writing good and catchy sentences, I should pay attention to the fact that non-native English speaker will read it too! Next time I'll try to use short, simple expression instead :smile:

File: pages/WhatIsOppia.html [hunk 13]
```diff
@@ -8,7 +8,7 @@
 
   <p>
     Oppia is an online learning tool that enables anyone to easily create and
-    share interactive activities ('explorations') which try to recreate the
+    share interactive activities (or <strong>explorations</strong>) which try to recreate the
```
Comment (seanlip) [inline pages/WhatIsOppia.html]: Reading this again, I think, maybe change 'or' to 'called'? Otherwise it reads as though explorations are an alternative to interactive activities, and that the two are mutually exclusive.

File: pages/home.html [hunk 14]
```diff
@@ -1,15 +1,20 @@
 <section id="main_content" class="inner">
 
+  <h2>
+    <a id="main-idea" class="anchor" href="#main-idea" aria-hidden="true">
```
Comment (seanlip) [inline pages/home.html]: Should the id and href be "motivation" instead?

File: pages/home.html [hunk 15]
```diff
@@ -62,15 +65,16 @@
...
-    <li>Allows learners to leave feedback on explorations.</li>
-    <li>Explorations are embeddable in any webpage.</li>
-    <li>Explorations can be played on mobile devices (except for some of the more complex interactions).</li>
-    <li>The exploration editor supports full version history, multiple authors, parameters and analytics dashboards.</li>
-    <li>There are extension frameworks for building new interactions and classification rules.</li>
-  </ul>
+  <p>With Oppia, you can:
+    <ul>
+      <li>simulate a 1-on-1 conversation with a tutor,</li>
+      <li>give targeted feedback to learners based on their answers (as a tutor),</li>
```
Comment (seanlip) [inline pages/home.html]: I still don't think "(as a tutor)" is very clear. Maybe just drop it?

File: pages/home.html [hunk 16]
```diff
@@ -62,15 +65,16 @@
...
+      <li>leave feedback on explorations (as a learner),</li>
```
Comment (seanlip) [inline pages/home.html]: "leave feedback on other people's explorations", or "leave feedback on explorations you have played"?

File: pages/home.html [hunk 17]
```diff
@@ -62,15 +65,16 @@
...
+      <li>embed explorations in any webpage,</li>
+      <li>play explorations on mobile devices,</li>
+      <li>have support on full version history (control of previous versions), multiple authors, parameters and analytics dashboards in the exploration editor.</li>
```
Comment (seanlip) [inline pages/home.html]: make use of full version history, ...

File: pages/home.html [hunk 18]
```diff
@@ -100,10 +104,7 @@
     Contributing to Oppia
   </h2>
   <p>
-    Oppia is built by the community for the community. Our mission is to enable
-    anyone to learn anything they want to in an effective, enjoyable way, and
-    building a platform that allows anyone to create and share bite-sized
-    interactive learning material is just a first step.
+    Built by the community for the community, Oppia helps anyone to learn anything they want to in an effective, enjoyable way. Building a platform that allows people to create and share bite-sized interactive learning material is just a first step.
```
Comment (seanlip) [inline pages/home.html]: This now feels like it's hanging. Perhaps:

Built by the community for the community, Oppia helps anyone to learn anything they want to in an effective, enjoyable way. So far, we've created a platform that allows people to create and share bite-sized interactive learning material, but this is just a first step.

File: pages/home.html (hunk 16 above)
Comment (MinhHuong) [inline pages/home.html]: maybe we can just let it be simple, "leave feedback on explorations", I think it's clear enough. The less we write the more readers can remember :smile:

**Comment by MinhHuong** [comment]
Hi @seanlip, I've just pushed a new commit that adjusts some minor changes as you've noted :smile:.

File: pages/TheExplorationEditor.html [hunk 19]
```diff
@@ -38,8 +38,7 @@
   </h3>
 
   <p>
-    A card consists of three parts: some 'content', an 'interaction' (or
-    question), and 'responses' that specify what Oppia should do next, based on
+    A card consists of three parts: some <strong>content</strong>, an <strong>interaction</strong> (or <strong>question</strong>), and <strong>responses</strong> that specify what Oppia should do next, based on
```
Comment (seanlip) [inline pages/TheExplorationEditor.html]: Sorry, I missed this: 'question' is not a term of art, so we probably don't need to make it bold.

File: pages/home.html [hunk 20]
```diff
@@ -1,15 +1,20 @@
 <section id="main_content" class="inner">
 
+  <h2>
+    <a id="main-idea" class="anchor" href="#motivation" aria-hidden="true">
```
Comment (seanlip) [inline pages/home.html]: Should the id be 'motivation', too?

File: pages/home.html [hunk 21]
```diff
@@ -100,10 +97,7 @@
     Contributing to Oppia
   </h2>
   <p>
-    Oppia is built by the community for the community. Our mission is to enable
-    anyone to learn anything they want to in an effective, enjoyable way, and
-    building a platform that allows anyone to create and share bite-sized
-    interactive learning material is just a first step.
+    Built by the community for the community, Oppia helps to learn anything you want to in an effective, enjoyable way. So far, we've developed a platform that allows people to create and share bite-sized interactive learning material, but this is just a first step. We're planning to add new features in order to bring you more interesting experiences with Oppia.
```
Comment (seanlip) [inline pages/home.html]: The first sentence of this is technically incorrect -- we aren't actually living up to that promise yet.

Perhaps "Oppia aims to help you learn ..."?

**Comment by seanlip** [comment]
Hi @MinhHuong -- thanks! I made a few more comments, but this looks almost ready.

Also, there are a bunch of other files [here](https://github.com/oppia/oppia.github.io/pull/31/commits/9938b49a601f08a622d6377f47d9257c95108fef) that seem to have been changed/added, but they probably shouldn't (at least not with this PR). Could you please revert that? Thanks!

File: pages/home.html [hunk 22]
```diff
@@ -21,20 +26,18 @@
...
-    webpage.
+    Explorations are easy to create and quick to play. They can be published in the exploration gallery, shared on social media, or embedded on your webpage.
   </p>
 
   <h3>
     <a id="interactive" class="anchor" href="#interactive" aria-hidden="true">
       <span class="octicon octicon-link"></span>
     </a>
-    Interactive
+    Interactive
```
Comment (seanlip) [inline pages/home.html]: Nit: There's an extra space at the end of this line that shouldn't be there.
